diff --git a/node_modules/@types/express/index.d.ts b/node_modules/@types/express/index.d.ts
index 787ec3a..8133575 100644
--- a/node_modules/@types/express/index.d.ts
+++ b/node_modules/@types/express/index.d.ts
@@ -47,7 +47,7 @@ declare namespace e {
      * This is a built-in middleware function in Express. It serves static files and is based on serve-static.
      */
-    var static: serveStatic.RequestHandlerConstructor<Response>;
+    var staticRequestHandler: serveStatic.RequestHandlerConstructor<Response>;
 
     /**
      * This is a built-in middleware function in Express. It parses incoming requests with urlencoded payloads and is based on body-parser.
